src/js/script.js

```diff
@@ -489,14 +497,43 @@
       const indexOfRemovedProduct = thisCart.products.indexOf(removedProduct);
       //console.log(indexOfRemovedProduct);
       const removedValues = thisCart.products.splice(indexOfRemovedProduct, 1);
-      //console.log(removedProduct);
-      //console.log(thisCart.product);
+      console.log(removedValues);
 
       removedProduct.dom.wrapper.remove();
 
       thisCart.update();
 
     }
+
+    sendOrder() {
+      const thisCart = this;
+      const url = settings.db.url + '/' + settings.db.orders;
+
+      const payload = {};
```

Comment:
Jeżeli jest to możliwe, to przy tworzeniu obiektu payload zainicjalizuj od razu jego wartości. Javascript na to pozwala, ale lepiej unikać sytuacji gdzie dynamicznie (w locie) tworzymy nowe pola. W tym przypadku, np. pole `payload.address` nie istnieje, a się do niego odwołujesz w linii 514.

Reply:
Jasne, poprawiłem.